code/modules/food_and_drinks/machinery/coffeemaker.dm

```diff
@@ -200,6 +273,24 @@
 		update_appearance()
 		return TRUE //no afterattack
 
+/obj/machinery/coffeemaker/proc/can_brew()
+	if(!cartridge)
+		balloon_alert_to_viewers("no coffee cartidge inserted!")
```

Comment:
Why are these sent to all viewers? You should probably be passing in a user that performed the action.

Reply:
An unfortunate blunder from me, changed to regular `balloon_alert()` in [1758d26](https://github.com/tgstation/tgstation/pull/70991/commits/1758d260bf38349a463fb0c4c94b19bde1396d9f)